app/static/css/core.css

```diff
@@ -162,7 +162,6 @@ Colour names courtesy of chir.ag/projects/name-that-color
 
 .btn {
   padding: 1rem;
-  width: 100%;
```

Comment:
This should not be removed as this affect also the terms of use buttons and the contextual help buttons.